packages/grpc-js-core/test/test-call-stream.ts

```diff
@@ -103,16 +103,16 @@ describe('CallStream', () => {
     assert2.afterMustCallsSatisfied(done);
   });
 
-  it('should end a call with an error if a stream was closed', (done) => {
+  describe('should end a call with an error if a stream was closed', () => {
     const c = http2.constants;
     const s = Status;
     const errorCodeMapping = {
-      [c.NGHTTP2_NO_ERROR]: s.INTERNAL,
+      [c.NGHTTP2_NO_ERROR]: s.OK,
       [c.NGHTTP2_PROTOCOL_ERROR]: s.INTERNAL,
       [c.NGHTTP2_INTERNAL_ERROR]: s.INTERNAL,
       [c.NGHTTP2_FLOW_CONTROL_ERROR]: s.INTERNAL,
       [c.NGHTTP2_SETTINGS_TIMEOUT]: s.INTERNAL,
-      [c.NGHTTP2_STREAM_CLOSED]: null,
+      [c.NGHTTP2_STREAM_CLOSED]: s.INTERNAL,
```

Comment:
The gRPC protocol spec doesn't specify a mapping for `STREAM_CLOSED`, so we shouldn't assert that the value is anything in particular in the tests. I think we should just skip this, at least for now.

Reply:
Done -- skipping this now.